Pull request

NewbieWanKenobi opened the pull request.

Commit:
Started stack.js

Commit:
Working through stack.js...

Commit:
JSX? Which JS version should I use to add methods to an object?

Commit:
Finished with stack.js

Commit:
Okay stack.js is offically done now!

Commit:
Started on linked-list.js

Commit:
Using my daughter's colored blocks with toothpicks to try to get this through my head.

Commit:
I feel like a feeble-minded primate.

Commit:
Learning by struggling--my daughter is looking at me like I'm crazy I've got all her blocks lined up with toothpicks linking them!

Commit:
Got linked list working! Now I'm going to try to make it a doubly-linked list.

JohnJustinn reviewed (commented):
This looks fantastic.  I can tell from the chain of commits that you really struggled through and learned while you worked through this sprint.   It is an impressive effort the work is very, very good.  Keep it up!

Commit:
Okay, hash table with objects as buckets seems to work fine but failing the following linter tests: ✕ should properly remove items (7ms)
    ✕ should not throw an error when attempting to retrieve a non-existent value (1ms)
    ✕ should not throw an error when removing a non-existent key (1ms)
